README-old.md

```diff
@@ -1,11 +1,13 @@
-# Docker
+# **** Disclaimer ****
```

Comment:
Please remove this old file and insert the old content in a confluence page so we can keep it for the future (instead looking on git)